lib/src/arithmetic/floating_point/floating_point_adder_singlepath.dart

```diff
@@ -353,6 +356,11 @@ class FloatingPointAdderSinglePath<FpTypeIn extends FloatingPoint,
       }
       exponentRound = exponent;
     }
+    // Handle Flush to Zero subnormal case
+    mantissaRound = (internalSum.subNormalAsZero
```

Comment:
what if `a` and `b` have different `subNormalAsZero`?  In this case, the output matches `a` if it's not provided.  That seems fine, but we should probably document that right?